tests/trigonometry.cpp

```diff
@@ -31,6 +35,19 @@ TEST(trigonometry, cos)
         auto cos_fixed = static_cast<double>(cos(P(flt_angle)));
         EXPECT_TRUE(HasMaximumError(cos_fixed, cos_real, MAX_ERROR_PERC));
     }
+
+    // Boundary-value analysis
+    constexpr int32_t raw_values[] = {INT32_MIN, 2147380703, 2147380704, INT32_MAX};
+    for (int32_t i=0; i<size(raw_values); i++)
+    {
+        constexpr auto MAX_ERROR_PERC = 0.034;  // ⚠️ Maximum deviation over the value range of f_16_16 --- despite the cosine bugfix
```

Comment:
> the cosine bugfix

Please describe the bug that was fixed for the benefit of future readers, they won't know what "the" bugfix means.
(same thing in the other test).

Reply:
Sorry, my ⚠️ note is a bit misleading. It was for you that the relative error as a test condition does not help. If you change your test loop to:

```cpp
for (int angle = -360*100; angle <= +360*100; ++angle)
    {
        auto flt_angle = angle * PI / 180 / 100;
```
the loop will end with fail, because the relative error in the value range is 4.92%.

I will commit a proposal later.